components/PsychrometricCharts.py

```diff
@@ -0,0 +1,177 @@
+    actual_vapor_pressure = relative_humidity / 100.0 * saturation_vapor_pressure
+    absolute_humidity = 216.7 * (actual_vapor_pressure / (dry_bulb_temp + 273.15))
+    return absolute_humidity
+
+def calculate_humidity_parameters(dry_bulb_temp, relative_humidity):
+    dew_point_temp = calculate_dew_point(dry_bulb_temp, relative_humidity)
+    wet_bulb_temp = calculate_wet_bulb_temp(dry_bulb_temp, relative_humidity)
+    absolute_humidity = calculate_absolute_humidity(dry_bulb_temp, relative_humidity)
+    enthalpy = (dry_bulb_temp + relative_humidity) * 0.24  
+    return {
+        "Dry Bulb Temperature (°C)": dry_bulb_temp,
+        "Relative Humidity (%)": relative_humidity,
+        "Absolute Humidity (g/m^3)": absolute_humidity,
+        "Wet Bulb Temperature (°C)": wet_bulb_temp,
+        "Dew Point Temperature (°C)": dew_point_temp,
+        "Enthalpy (kJ/kg)": enthalpy  
+    }
+
+def generate_vapor_pressure_chart(fixed_temperature, fixed_humidity):
+    temperatures = np.linspace(10, 36, 500)
+    relative_humidities = np.array([10, 20, 30, 40, 50, 60, 70, 80, 90, 100])
+    
+    traces = []
+
+
+    green_zone_1_left = 22
+    green_zone_1_right = 24
+    green_zone_2_left = 24
+    green_zone_2_right = 26
+    green_zone_3_left = 26
+    green_zone_3_right = 28
+
+    vapor_pressure_min = 0  
+    vapor_pressure_max = 3000
+
+    traces.append(go.Scatter(
+        x=np.concatenate([np.array([green_zone_1_left, green_zone_1_right]), 
+                          np.array([green_zone_1_right, green_zone_1_left])]),
+        y=np.concatenate([np.array([vapor_pressure_max, vapor_pressure_max]), 
+                          np.array([vapor_pressure_min, vapor_pressure_min])]),
+        fill='toself',
+        fillcolor='rgba(0, 255, 0, 0.2)',  
+        line=dict(color='rgba(255,255,255,0)'),
+        showlegend=False
+    ))
+
+    traces.append(go.Scatter(
+        x=np.concatenate([np.array([green_zone_2_left, green_zone_2_right]), 
+                          np.array([green_zone_2_right, green_zone_2_left])]),
+        y=np.concatenate([np.array([vapor_pressure_max, vapor_pressure_max]), 
+                          np.array([vapor_pressure_min, vapor_pressure_min])]),
+        fill='toself',
+        fillcolor='rgba(0, 255, 0, 0.4)',  
+        line=dict(color='rgba(255,255,255,0)'),
+        showlegend=False
+    ))
+
+    traces.append(go.Scatter(
+        x=np.concatenate([np.array([green_zone_3_left, green_zone_3_right]), 
+                          np.array([green_zone_3_right, green_zone_3_left])]),
+        y=np.concatenate([np.array([vapor_pressure_max, vapor_pressure_max]), 
+                          np.array([vapor_pressure_min, vapor_pressure_min])]),
+        fill='toself',
+        fillcolor='rgba(0, 255, 0, 0.6)',  
+        line=dict(color='rgba(255,255,255,0)'),
+        showlegend=False
+    ))
+
+    
+    for RH in relative_humidities:
```

Comment:
use PEP naming convention, RH should be in lower case

Reply:
got it , i already make them same name as the prototype

Reply:
![image](https://github.com/user-attachments/assets/299113d0-2f2d-4875-b236-8b01a45525e5)
i already make RH in lower case